hat/app/org/hatdex/hat/utils/Mailer.scala

```diff
@@ -137,14 +145,14 @@ class HatMailerImpl @Inject() (
   def serverErrorNotify(
       request: RequestHeader,
       exception: UsefulException
-    )(implicit m: Messages): Done = {
+    )(implicit m: Messages): Unit = {
     sendEmail(adminEmails: _*)(
       from = emailFrom,
       subject = s"HAT server ${request.host} error #${exception.id}",
       bodyHtml = views.html.mails.emailServerError(request, exception).toString(),
       bodyText = views.html.mails.emailServerError(request, exception).toString()
     )
-    Done
+
```

Comment:
@xavji Would like this be better?

Reply:
no you are still discarding the result of `sendEmail`.
either we `Await.result(sendEmai(...), someDuration)` or the methods return the `Future[Done]` from `sendEmail`